requirements_dev.txt

```diff
@@ -7,6 +7,8 @@ tox==3.14.0
 coverage==4.5.4
 Sphinx==1.8.5
 twine==1.14.0
+pandas==1.5.3
```

Comment:
Please add those dependencies also into requirements = ['pandas', 'numpy',] in setup.py.

For more information, see https://stackoverflow.com/a/49684835